keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md

```diff
@@ -380,112 +408,106 @@ _This section must be completed when targeting alpha to a release._
     - Will enabling / disabling the feature require downtime or reprovisioning
       of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
-* **Does enabling the feature change any default behavior?**
+#### Does enabling the feature change any default behavior?
 
   Yes.
   
   - Removing terminated Pods doesn't affect Job status.
   - Pods removed by the user or other controllers count towards failures or
     completions.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?**
+#### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
   
   Yes.
   The job controller removes finalizers in this case.
   Since some succeeded Pods might have been removed, the job controller will
   create new Pods to fulfill completions. But this is no different from
   existing behavior with the legacy tracking.
 
-* **What happens if we reenable the feature if it was previously rolled back?**
+#### What happens if we reenable the feature if it was previously rolled back?**
 
   Existing Jobs are tracked with legacy Pods.
 
-* **Are there any tests for feature enablement/disablement?**
+#### Are there any tests for feature enablement/disablement?**
```

Comment:
Have those been added? If not, can we add a similar comment as for upgrade tests?

Reply:
Yes, we have them. Updated.

Reply:
Can you please add links?

Reply:
Done :)